addon-resizer/nanny/estimator.go

```diff
@@ -43,19 +43,28 @@ func (e LinearEstimator) scaleWithNodes(numNodes uint64) *corev1.ResourceRequire
 	return calculateResources(numNodes, e.Resources)
 }
 
-// ExponentialEstimator estimates the amount of resources in the way that
-// prevents from frequent updates but may end up with larger resource usage
-// than actually needed (though no more than ScaleFactor).
+// ExponentialEstimator estimates resource requirements in a way that prevents
+// frequent updates but may end up with larger estimates than actually needed.
 type ExponentialEstimator struct {
-	Resources   []Resource
+	// The collection of resources to provide estimates for.
+	Resources []Resource
+	// The minimum cluster size for which the estimator will provide
+	// resource estimates.
+	MinClusterSize uint64
+	// The multiplier used to compute the next cluster size to provide
+	// estimates for. For example, suppose the cluster has 9 nodes,
+	// MinClusterSize is 5, and ScaleFactor is 1.5. Then the estimator will
+	// provide estimates for FLOOR(5 * 1.5 * 1.5) = 11 nodes since that is
+	// the smallest cluster size larger than 9 nodes.
 	ScaleFactor float64
 }
 
 func (e ExponentialEstimator) scaleWithNodes(numNodes uint64) *corev1.ResourceRequirements {
-	n := uint64(16)
+	n := e.MinClusterSize
```

Comment:
With this change, if anyone passes 0 or 1 as a flag value, this will go into infinite loop. Probably not what one might expect.

Reply:
Done.